pkg/pb/satellite.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package satellite;
+
+import "gogo.proto";
+
+service Satellite {
+    rpc Health(FileHealthRequest) returns (FileHealthResponse) {}
+}
+  
+message FileHealthRequest {
+    string path = 1;
+    bytes uplink_id = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];    // Uplink Identity
+}
+
+message FileHealthResponse {
+    message SegmentInfo{
+        int64 good_nodes = 1; // How many pieces are currently on good nodes
+        int64 bad_nodes = 2; // How many pieces are currently on bad nodes
+        int64 offline_nodes = 3; // How many pieces are currently on offline nodes
+        int64 below_recover = 4; // How many pieces are needed to recover
+        int64 below_repair = 5; // How many pieces are needed for being above the repair threshold
+        int64 below_success = 6; // How many pieces are needed to be at the success threshold
+    }
+
+    repeated SegmentInfo segments = 1;
```

Comment:
Do we care about per-segment info? I think total over the full file seems sufficient.

Reply:
I think we can do more with per-segment info. Also if we just list everything together we won't know if we have bad segments or not